internal/manifests/config/tempo-config.yaml

```diff
@@ -30,6 +51,13 @@ distributor:
             key_file: {{ .TLS.Paths.Key }}
             min_version: {{ .TLS.Profile.MinTLSVersionShort }}
 {{- end }}
+{{- if .ReceiverTLS.Enabled }}
+          tls:
+            ca_file:  {{ .ReceiverTLS.Paths.CA }}
+            cert_file: {{ .ReceiverTLS.Paths.Certificate }}
+            key_file: {{ .ReceiverTLS.Paths.Key }}
+            min_version: {{ .ReceiverTLS.MinTLSVersion }}
+{{- end }}
 {{- if not .Gateway }}
         http:
           endpoint: 0.0.0.0:4318
```

Comment:
OTLP/HTTP also supports TLS according to https://grafana.com/docs/tempo/latest/configuration/tls/